### PR TITLE
feat(cni): add tunnel port in init_agent

### PR DESCRIPTION
**Pull request by qingwusunny**

### CODE REVIEW

File: build/images/release/Dockerfile [hunk 1]
```diff
@@ -25,10 +25,16 @@ RUN chmod 755 /opt/everoute/bin/*
 
 #FROM ubuntu
 FROM ubuntu:20.04
+ARG TARGETOS
+ARG TARGETARCH
```
Comment (wu-cl) [inline build/images/release/Dockerfile:29]: duplicate with L11, or is it diff after FROM?

File: build/images/release/Dockerfile [hunk 2]
```diff
@@ -25,10 +25,16 @@ RUN chmod 755 /opt/everoute/bin/*
...
+RUN echo "platform $TARGETOS $TARGETARCH"
 
-#RUN apk update && apk add openvswitch
+#RUN apt update && apt add openvswitch
```
Comment (wu-cl) [inline build/images/release/Dockerfile:32]: just remove this line

File: build/images/release/Dockerfile [hunk 3]
```diff
@@ -25,10 +25,16 @@ RUN chmod 755 /opt/everoute/bin/*
...
 RUN apt update && apt install -y openvswitch-switch=2.13.* iptables \
-    conntrack=1:1.4.5-2 iproute2 tcpdump && rm -rf /var/lib/apt/lists/*
+    conntrack=1:1.4.5-2 iproute2 tcpdump wget&& rm -rf /var/lib/apt/lists/*
```
Comment (wu-cl) [inline build/images/release/Dockerfile:34]: add SPACE before &&

**Review by wu-cl — APPROVED** [review]

File: build/images/release/Dockerfile (hunk 1 above)
Comment (qingwusunny) [reply build/images/release/Dockerfile:29]: 在From之前定义的ARG不能在From之后使用。测试也是不加的话，后面读不出来这个值

**Review by zwtop — APPROVED** [review]

**Pull request merged**